python/python/tests/test_async_client.py

```diff
@@ -1063,6 +1063,144 @@ async def test_linsert(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.linsert(key2, InsertPosition.AFTER, "p", "e")
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_lmove(self, redis_client: TRedisClient):
+        key1 = "{SameSlot}" + get_random_string(10)
+        key2 = "{SameSlot}" + get_random_string(10)
+
```

Comment:
nit: check redis version in the beginning of the test

Reply:
Why?
We aren't going to start testing valkey version less than 6.2.0.

Reply:
Available since:
7.0.0